src/sc_pipe_management/analysis_set_setup_utils.py

```diff
@@ -27,8 +27,8 @@ def get_parser():
                         help="""Lab ID for the analysis set.""")
     parser.add_argument('--award', type=str,
                         help="""Award ID for the analysis set.""")
-    parser.add_argument('--preferred_assay_title', type=str,
-                        help="""Preferred assay title for the measurement set query.""")
+    parser.add_argument('--preferred_assay_titles', type=str,
```

Comment:
I think I missed this previously, should use `nargs='*'` or `nargs="+"` if you want to pass a list of items.